### DISCUSSION TITLE
Training dreambooth in runpod and using that model with control net functions

### DISCUSSION THREAD

**Discussion by vitalijmirnyj** [TheLastBen/fast-stable-diffusion#2341] [Q&A]
Hello. I have a question about runpod and dreambooth. I want to train my own model in runpod and somehow use that model with control net. Is that possible? I know how to use runpod and train my own model there. But have no idea how to use that with control net. Could someone please tell me how to do that if that is possible? :) Thank you

**Comment by TheLastBen** [comment]
First open the A1111 notebook, run all the cell until the controlnet cell, after that, go back to the dreambooth notebook, run the first cell, then run the testing cell

**Reply by vitalijmirnyj** [reply]
Thank you so much. I do not know how, but it works :)

**Reply by TheLastBen** [reply]
great!
